<launch>

    <node name="estimator" pkg="rotors_exercise" type="estimator_node" clear_params="true" output="screen">

    <!--
       <remap from="/firefly/imu" to="/firefly/imu" />  
       <remap from="/firefly/fake_gps/pose_with_covariance" to="/firefly/fake_gps/pose_with_covariance" />
       <remap from="/firefly/pose_with_covariance" to="/firefly/pose_with_covariance" />
   	-->

    </node>
</launch>
